src/chatlist/chatList.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { withStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import Avatar from "@material-ui/core/Avatar";
+import Typography from "@material-ui/core/Typography";
+import styles from "./styles";
+import Divider from "@material-ui/core/Divider";
+import Button from "@material-ui/core/Button";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import NotificationImportant from "@material-ui/icons/NotificationImportant";
+
+class ChatListComponent extends React.Component {
+  newChat = () => {
+    console.log("New chat click");
+  };
+
+  selectChat = (index) => {
+    console.log("select chat", index);
+  };
+  render() {
+    const { classes } = this.props;
+
+    if (this.props.chats.length > 0) {
+      return (
+        <main className={classes.root}>
+          <Button
+            variant="contained"
+            fullWidth
+            color="primary"
+            className={classes.newChatBtn}
+            onClick={this.newChat}
+          >
+            New Message
+          </Button>
+          <List>
+            {this.props.chats.map((_chat, _index) => {
+              return (
+                <div key={_index}>
+                  <ListItem
+                    onClick={() => this.selectChat(_index)}
+                    className={classes.listItem}
+                    selected={this.props.selectedChatIndex === _index}
+                    alignItems="flex-start"
+                  >
+                    <ListItemAvatar>
+                      <Avatar alt="Remy Sharp">
+                        {
+                          _chat.users
+                            .filter(
+                              (_user) => _user !== this.props.userEmail
+                            )[0]
+                            .split("")[0]
+                        }
+                      </Avatar>
+                    </ListItemAvatar>
+                    <ListItemText
+                      primary={
+                        _chat.users.filter(
+                          (_user) => _user !== this.props.userEmail
```

Comment:
It looks like 
```
chat.users
  .filter(
    (_user) => _user !== this.props.userEmail
  )[0]
```
is repeated. Maybe consider moving it into a variable, so it only needs to be written once?

I assume that this line `.split("")[0]` is for splitting the user's name, and grabbing just the first name? If so, pop it in a comment!

Reply:
I am not sure where to put the variable:(
`.split("")[0]` is for splitting the user's name, and grabbing just the first letter to show in Avatar.